metal/tuners/tuner.py

```diff
@@ -80,7 +80,7 @@ def _clear_state(self):
         self.start_time = time()
         self.run_stats = []
         self.best_index = -1
-        self.best_score = -1
+        self.best_score = -1e9
```

Comment:
What's the motivation for this change?

Reply:
Negative loss can be < -1, so makes sure we update.